src/v2/view-builder/views/TerminalView.js

```diff
@@ -12,24 +12,35 @@ const OPERATION_CANCELED_BY_USER_KEY = 'idx.operation.cancelled.by.user';
 const DEVICE_ACTIVATED = 'idx.device.activated';
 const DEVICE_NOT_ACTIVATED_CONSENT_DENIED = 'idx.device.not.activated.consent.denied';
 const DEVICE_NOT_ACTIVATED_INTERNAL_ERROR = 'idx.device.not.activated.internal.error';
+const FLOW_CONTINUE_IN_NEW_TAB = 'idx.transferred.to.new.tab';
+const EMAIL_LINK_OUT_OF_DATE = 'idx.return.stale';
+const EMAIL_LINK_CANT_BE_PROCESSED = 'idx.return.error';
+const EMAIL_FINISH_SIGN_IN = 'idx.email.verification.required';
 
 export const REGISTRATION_NOT_ENABLED = 'oie.registration.is.not.enabled';
 export const FORGOT_PASSWORD_NOT_ENABLED = 'oie.forgot.password.is.not.enabled';
 
 const EMAIL_AUTHENTICATOR_TERMINAL_KEYS = [
-  'idx.transferred.to.new.tab',
-  'idx.return.stale',
-  'idx.return.error',
-  'idx.email.verification.required',
+  FLOW_CONTINUE_IN_NEW_TAB,
+  EMAIL_LINK_OUT_OF_DATE,
+  EMAIL_LINK_CANT_BE_PROCESSED,
+  EMAIL_FINISH_SIGN_IN,
   RETURN_TO_ORIGINAL_TAB_KEY,
   RETURN_LINK_EXPIRED_KEY,
   OPERATION_CANCELED_ON_OTHER_DEVICE_KEY,
   OPERATION_CANCELED_BY_USER_KEY,
 ];
 
-const GET_BACK_TO_SIGN_LINK_FLOWS = [
-  RETURN_LINK_EXPIRED_KEY,
-  REGISTRATION_NOT_ENABLED,
+// These terminal views build their own links, basically they have cancel remediation in error response
+// Or doesn't require a Back to Sign in link because the flow didn't start from login screen
+const NO_BACKTOSIGNIN_LINK_VIEWS = [
+  UNLOCK_ACCOUNT_TERMINAL_KEY,
+  RETURN_TO_ORIGINAL_TAB_KEY,
+  FLOW_CONTINUE_IN_NEW_TAB,
```

Comment:
Do we need `EMAIL_LINK_OUT_OF_DATE` & `EMAIL_LINK_CANT_BE_PROCESSED` added here? and how about `EMAIL_FINISH_SIGN_IN` (I don't know under what scenario this may occur.)?

Reply:
I added these but it breaks testcafe tests means those flows expected the `Back to sign in` link